packages/lesswrong/components/ea-forum/TargetedJobAd.tsx

```diff
@@ -0,0 +1,377 @@
+    flash({messageString: "Thanks for registering interest!", type: "success"})
+  }
+  
+  const { HoverPreviewLink, LWTooltip } = Components
+  
+  if (loading || (results?.length && results[0].interestedInMetaculus) || cookies[HIDE_JOB_AD_COOKIE] || !showJobAd) {
+    return null
+  }
+
+  return <div className={classes.root}>
+      <img src="https://80000hours.org/wp-content/uploads/2019/07/Metaculus-160x160.jpeg" className={classes.logo} />
+      <div className={classes.bodyCol}>
+        <div className={classes.topRow}>
+          <div className={classes.label}>
+            <div className={classes.labelText}>
+              Job  recommendation
+            </div>
+            <LWTooltip title="You're seeing this recommendation because of your interest in software engineering. We encourage you to consider jobs like this which might increase your impact significantly.">
+              <InfoIcon className={classes.infoIcon} />
+            </LWTooltip>
+          </div>
+          <div className={classes.feedbackLink}>
+            <a href="https://docs.google.com/forms/d/e/1FAIpQLSdGyKmZRZHqdhEc70QNIzOTKy_j1aMEByGhE_HtciSNMUSJTA/viewform" target="_blank" rel="noopener noreferrer">
+              Give us feedback on this experiment
+            </a>
+          </div>
+          <Tooltip title="Dismiss">
+            <Button className={classes.closeButton} onClick={dismissJobAd}>
+              <CloseIcon className={classes.closeIcon} />
+            </Button>
+          </Tooltip>
+        </div>
+        <h2 className={classes.header}>
+          {/* Direct link to job description: https://apply.workable.com/metaculus/j/409AECAA94/ */}
+          <a href="https://efctv.org/3A16UNq" target="_blank" rel="noopener noreferrer" className={classes.link}>
+            Full-stack engineer
+          </a> at <span className={classes.link}>
+            <HoverPreviewLink href="/topics/metaculus" innerHTML="Metaculus" />
+          </span>
+        </h2>
+        <div className={classes.metadataRow}>
+          <div className={classes.metadata}>
+            $70k - $120k
+          </div>
+          <div className={classes.metadata}>
+            <LocationIcon className={classes.metadataIcon} />
+            Remote
+          </div>
+        </div>
+        {!expanded && <button onClick={handleReadMore} className={classes.readMore}>Read more</button>}
+        
+        {expanded && <>
+          <div className={classes.description}>
+            <a href="https://www.metaculus.com" target="_blank" rel="noopener noreferrer" className={classes.link}>
+              Metaculus
+            </a> is an online forecasting platform working to improve human reasoning and coordination on topics
+            of global importance.
+          </div>
+          <div className={classes.description}>
+            Ideal candidates:
+            <ul>
+              <li>Have experience shipping features end-to-end (CSS, Angular/React, API, & Python/Django)</li>
```

Comment:
This is super nit-picky, but I think the Oxford comma should be omited with an ampersand